parlai/crowdsourcing/utils/acceptability.py

```diff
@@ -11,15 +11,17 @@
 
 
 class AcceptabilityChecker:
+
+    ALL_VIOLATION_TYPES = [
```

Comment:
ditto with this i guess - is there a way to add this as a cmd line arg instead of having to manually re-write the file?

Reply:
hmm, here this is a list of all possible violation types supported by `AcceptabilityChecker` given its inherent functionality. Anyone wishing to add others would have to subclass the checker, and then they can modify `ALL_VIOLATION_TYPES`